src/test/load-addressbook.spec.js

```diff
@@ -0,0 +1,28 @@
+import { Selector } from "testcafe";
+
+fixture("Load AddressBook").page`http://localhost:3000`;
+
+const Document = "./fixture/maersk-bill-of-lading.tt";
+const AddressBook = "./fixture/local-addressbook.csv";
+
+const TitleTransferPanel = Selector("#title-transfer-panel");
+const StatusButton = Selector("#certificate-status");
+const BeneficiaryName = Selector("#beneficiary h5");
+const HolderName = Selector("#holder h5");
+const CSVFileInput = Selector("#csv-file-input");
+
+const validateTextContent = async (t, component, texts) =>
+  texts.reduce(async (prev, curr) => t.expect(component.textContent).contains(curr), Promise.resolve());
+
+test("AddressBook local names to be resolved correctly", async t => {
+  await t.setFilesToUpload("input[type=file]", [Document]);
+  await TitleTransferPanel.with({ visibilityCheck: true })();
+  await validateTextContent(t, StatusButton, ["Issued by TRADETRUST.IO"]);
+
+  const elExists = await CSVFileInput.exists;
+  if (elExists) {
```

Comment:
why only if csv input exist? if it's because of the env?

Reply:
yeah, passed in local but failing here. i tried timeout also cant work. putting the exists somehow passed here

Reply:
wait, then i couldn't tell too if the csvfile upload test is working here, lol

Reply:
yea, if its not running the test then it will just fail silently, you want to find out why it fails. Is it because the feature is hidden away?

Reply:
oh ya i think so, the feature toggle. let me try enable it

Reply:
yeah, enable passed liao